osu.Game/FodyWeavers.xsd

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <!-- This file was generated by Fody. Manual changes to this file will be lost when your project is rebuilt. -->
```

Comment:
Is this file required or can it be .gitignored?

Reply:
Pretty sure we want it committed. It allows for configuration to be provided, if we ever need that.

Reply:
This comment indicates you shouldn't be changing this file though.

Reply:
Was referring to the other one, but most other projects look to commit this https://www.google.com/search?client=firefox-b-d&q=FodyWeavers.xsd+